recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+      "2023-02-23 09:55:43.582 | DEBUG    | pint.toa:__init__:1327 - No pulse number flags found in the TOAs\n",
+      "2023-02-23 09:55:43.583 | DEBUG    | pint.toa:apply_clock_corrections:2132 - Applying clock corrections (include_gps = False, include_bipm = False)\n",
+      "2023-02-23 09:55:43.589 | DEBUG    | pint.toa:compute_TDBs:2182 - Computing TDB columns.\n",
+      "2023-02-23 09:55:43.590 | DEBUG    | pint.toa:compute_TDBs:2207 - Using EPHEM = DE421 for TDB calculation.\n",
+      "2023-02-23 09:55:43.595 | DEBUG    | pint.toa:compute_posvels:2291 - Computing PosVels of observatories, Earth and planets, using DE421\n",
+      "2023-02-23 09:55:44.419 | INFO     | pint.solar_system_ephemerides:_load_kernel_link:53 - Set solar system ephemeris to de421 from download\n",
+      "2023-02-23 09:55:44.431 | DEBUG    | pint.toa:compute_posvels:2353 - SSB obs pos [-1.49278181e+08  7.07659442e+06  3.07113250e+06] km\n",
+      "2023-02-23 09:55:45.436 | INFO     | pint.solar_system_ephemerides:_load_kernel_link:53 - Set solar system ephemeris to de421 from download\n",
+      "2023-02-23 09:55:46.473 | INFO     | pint.solar_system_ephemerides:_load_kernel_link:53 - Set solar system ephemeris to de421 from download\n",
+      "2023-02-23 09:55:47.333 | INFO     | pint.solar_system_ephemerides:_load_kernel_link:53 - Set solar system ephemeris to de421 from download\n",
+      "2023-02-23 09:55:48.649 | INFO     | pint.solar_system_ephemerides:_load_kernel_link:53 - Set solar system ephemeris to de421 from download\n",
+      "2023-02-23 09:55:49.572 | INFO     | pint.solar_system_ephemerides:_load_kernel_link:53 - Set solar system ephemeris to de421 from download\n",
+      "2023-02-23 09:55:50.378 | INFO     | pint.solar_system_ephemerides:_load_kernel_link:53 - Set solar system ephemeris to de421 from download\n",
+      "2023-02-23 09:55:51.223 | INFO     | pint.solar_system_ephemerides:_load_kernel_link:53 - Set solar system ephemeris to de421 from download\n",
+      "2023-02-23 09:55:51.237 | DEBUG    | pint.toa:compute_posvels:2367 - Adding columns ssb_obs_pos ssb_obs_vel obs_sun_pos obs_jupiter_pos obs_saturn_pos obs_venus_pos obs_uranus_pos obs_neptune_pos obs_earth_pos\n",
+      "2023-02-23 09:55:51.240 | DEBUG    | pint.models.absolute_phase:get_TZR_toa:110 - Done with TZR_toa\n",
+      "2023-02-23 09:55:51.309 | DEBUG    | pint.models.glitch:glitch_phase:212 - Glitch phase for glitch 1: 0.0 \n"
+     ]
+    }
+   ],
+   "source": [
+    "# compute phases\n",
+    "phases = model.phase(ts,abs_phase=True)[1]\n",
+    "                \n",
+    "# shift phases to the interval (0,1]\n",
+    "phases = np.where(phases < 0.0 , phases + 1.0 , phases)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "466a291e",
+   "metadata": {},
+   "source": [
+    "## 3. Adding the phases to an EvenList and put it in a new Observation. "
+   ]
```

Comment:
Add some lines about why it is necessary to create a new eventlist and a new observation